umrr_ros2_driver/include/umrr_ros2_driver/smartmicro_radar_node.hpp

```diff
@@ -82,6 +89,31 @@ class UMRR_ROS2_DRIVER_PUBLIC SmartmicroRadarNode : public ::rclcpp::Node
   ///
   void update_config_files_from_params();
 
+   ///
+  /// @brief      Callaback for getting the instruction response.
+  ///
+  void MyResponseCallback(const ClientId,const std::shared_ptr<com::master::ResponseBatch> & response);
```

Comment:
Mixing case is bad practice and leads to messy code. Please use the same case as other functions in this class. e.g. `my_response_callback(const client_id, const ...)`).